src/main/scala/zio/prelude/GenFs.scala

```diff
@@ -59,6 +81,15 @@ object GenFs {
         Gens.nonEmptyListOf(gen)
     }
 
+  /**
+   * A generator of `Try` values.
+   */
+  val `try`: GenF[Random with Sized, Try] =
+    new GenF[Random with Sized, Try] {
+      def apply[R1 <: Random with Sized, A](gen: Gen[R1, A]): Gen[R1, Try[A]] =
+        oneOf(Gen.throwable.map(scala.util.Failure(_)), gen.map(scala.util.Success(_)))
```

Comment:
I had to do manually write a `Gen` for `Try` because it seems there isn't a `Gen.try` in zio-test